### ISSUE TITLE
[Bug] gpt说不完话，或重复一段话

### ISSUE THREAD

**Issue by Sha1rholder** [Yidadaa/ChatGPT-Next-Web#1579]
当我在使用gpt4时，不确定是由于我的网络问题还是其他问题，总是回答不完，且后续不会再补完内容：（基本排除是gpt本身就要这么回答）
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/96712109/42edb3c1-0f45-441f-ac1f-c15c51f3fb62)
有时候它甚至重复一段内容：（基本排除是gpt本身就要这么回答）
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/96712109/6eb691dd-0918-43e2-b112-806c9ccdda54)
这是网络问题导致的吗？ 有什么可以优化的吗？

 - Deployed in Vercel
 - OS: Windows11
 - Browser: Edge (download to app)
 - Version: newest

**Comment by Sha1rholder** [comment]
> Could you please check if the commit have merge to your respository? [db29fed](https://github.com/Yidadaa/ChatGPT-Next-Web/commit/db29fed99ac2314b3c9fa5a209aa1565598d683b)

No, I haven't. I'll try it now, thx. The problem is that I can't reproduce this bug (it appeared in a crowded web environment) so I'm not sure if this bug has been fixed by new merge.

**Comment by Yidadaa** [comment]
手动更新到最新版本的代码，今天刚修复这个问题。
